File: BiosenseSignal_Flutter_SDK_5.6.3/ios/Classes/NativeBridgeApi.swift
//
//  NativeBridgeApi.swift
//  biosensesignal_flutter_sdk
//
//  Created on 26/03/2023.
//

import Foundation


class NativeBridgeApi {
    
    static let createSession = "createSession";
    static let createPPGDeviceSession = "createPPGDeviceSession"
    static let startSession = "startSession";
    static let stopSession = "stopSession";
    static let terminateSession = "terminateSession";
    static let getSessionState = "getSessionState";
    static let getNativeSdkVersion = "getNativeSdkVersion";
    static let getPolarMinVersion = "getPolarMinVersion";
    static let startPPGDevicesScan = "startPPGDevicesScan";
    static let stopPPGDeviceScan = "stopPPGDevicesScan";
}

File: BiosenseSignal_Flutter_SDK_5.6.3/android/src/main/kotlin/com/biosensesignal/flutter_sdk/NativeBridgeApi.kt
package com.biosensesignal.flutter_sdk

abstract class NativeBridgeApi {

    companion object {
        const val createSession = "createSession";
        const val createPPGDeviceSession = "createPPGDeviceSession";
        const val startSession = "startSession";
        const val stopSession = "stopSession";
        const val terminateSession = "terminateSession";
        const val getSessionState = "getSessionState";
        const val getNativeSdkVersion = "getNativeSdkVersion";
        const val getMinPolarVersion = "getMinPolarVersion";
        const val startPPGDevicesScan = "startPPGDevicesScan";
        const val stopPPGDeviceScan = "stopPPGDevicesScan";
    }
}